src/components/Register.vue

```diff
@@ -1,5 +1,44 @@
 <template>
-  
+  <!DOCTYPE html>
+<html lang="en" class="h-full">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Registration</title>
+    <link href="https://unpkg.com/tailwindcss@1.1.4/dist/tailwind.min.css" rel="stylesheet" />
+  </head>
```

Comment:
1. 1. Remove these html and head tags(Template tag doesn't have that)
2. Remove Doc Type